### DISCUSSION TITLE
Issue found on line 12454 of MergedAllCoreOntology-v1.5-2024-02-14.ttl

### DISCUSSION THREAD

**Discussion by sgranite** [CommonCoreOntology/CommonCoreOntologies#374] [Announcements]
The cco:definition_source on line 12454 has a type of xsd:anyURI.  However the string for the definition_source reads "https://oceanservice.noaa.gov/facts/cryosphere.html (accessed 03/06/2023)".  The "(accessed 03/06/2023)" portion of the value needs to be removed in order for the ontology to not experience a parsing error.

**Comment by swartik** [comment]
Actually, that may be valid. From the definition of [xsd:anyURI](https://www.w3.org/TR/xmlschema11-2/#anyURI):

    anyURI represents an Internationalized Resource Identifier Reference (IRI) ... as defined in RFC 3987 ...

From RFC 3987:

    Systems accepting IRIs MAY also deal with the printable characters in
    US-ASCII that are not allowed in URIs, namely "<", ">", '"', space,
    "{", "}", "|", "\", "^", and "`", in step 2 above.

So it shouldn't necessarily yield a parsing error. But of course any program blindly fetching that URI literal is going to receive a 404. For what it's worth, Protege is smart enough to create a hyperlink only for the URL portion.

**Comment by cameronmore** [comment]
Thank you for pointing this out. This warrants including as a policy whether the accessed date should be preserved in another annotation, or the datatype be changed, or access date dropped.

Looking more broadly, the only other terms that mention the accessed date are dc terms bibliographic citation, in which the access date is of course warranted. This is the only case where an access date is used with the annotation cco:definition_source.

This also raises the issue of making sure that external URIs are routinely checked to make sure no terms point to dead links or links to webpages which have changed that do not reflect the definition.

@mark-jensen @johnbeve @mark-jensen

**Comment by cameronmore** [comment]
As we improve definition sources, it is also under discussion what value we should have to our definition source annotation properties, and this will be clarified and fixed when that occurs.
